test/org/sagebionetworks/bridge/services/HealthCodeServiceTest.java

```diff
@@ -24,6 +34,16 @@
     @Resource
     private HealthCodeService healthCodeService;
 
+    @Before
+    public void before() {
+        clearDynamo();
+    }
+
+    @After
+    public void after() {
+        clearDynamo();
+    }
```

Comment:
There is no need to clear dynamo before and after each test run. The test runs fine with other entries in the table.

Clearing Dynamo means our local test accounts lose their healthcodes, which makes local testing a pain.